en/download.rst

```diff
@@ -6,33 +6,31 @@
 Download
 ================================================================================
 
-The latest stable release of |osgeo-live-version| :
+The latest stable release is |osgeo-live-version| and can be downloaded from:
+https://sourceforge.net/projects/osgeo-live/files/
 
-.. image:: ../images/download_buttons/download-dvd.png
-  :alt: Download 32 bit iso file
-  :align: left
-  :target: http://download.osgeo.org/livedvd/release/9.5/osgeo-live-9.5-i386.iso/download
+Picking the right image for you:
 
-`osgeo-live-9.5-i386.iso <http://download.osgeo.org/livedvd/release/9.5/osgeo-live-9.5-i386.iso/download>`_ : |osgeolive-iso-size|
+**osgeo-live iso**: |osgeolive-iso-size|
 
-.. image:: ../images/download_buttons/download-dvd.png
-  :alt: Download 64 bit iso file
-  :align: left
-  :target: http://download.osgeo.org/livedvd/release/9.5/osgeo-live-9.5-amd64.iso/download
+A Lubuntu based bootable image, which can be copied to an 8 Gig USB thumb drive (faster and recommended) or DVD (cheaper). This image includes windows and mac installers for many of the applications, and is the image you would typically use if handing out OSGeo-Live to new users at a conference.
 
-`osgeo-live-9.5-amd64.iso <http://download.osgeo.org/livedvd/release/9.5/osgeo-live-9.5-amd64.iso/download>`_ : |osgeolive-iso-size|
+**osgeo-live-mini iso**: |osgeolive-iso-mini-size|
```

Comment:
There is no mini iso anymore, is has been renamed to the main iso and it does not include the mac and win installers. This is what we decided some months ago.